### PR TITLE
Add r-heatmap.plus.

### PR DESCRIPTION
**Pull request by cbrueffer**

### CODE REVIEW

File: recipes/r-heatmap.plus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  commands:
+    - $R -e "library('heatmap.plus')"  # [not win]
+    - "\"%R%\" -e \"library('heatmap.plus')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=heatmap.plus
+  license: GPL
+  summary: Allows heatmap matrix to have non-identical X- and Y-dimensions.  Allows multiple
+    tracks of annotation for RowSideColors and ColSideColors.
+  license_family: GPL
```
Comment (bgruening) [inline recipes/r-heatmap.plus/meta.yaml]: the license_file is here missing

Comment (cbrueffer) [reply recipes/r-heatmap.plus/meta.yaml]: Well, it's just licensed under "GPL", so it's not really clear which one to pick.

Comment (bgruening) [reply recipes/r-heatmap.plus/meta.yaml]: Is there a file included in the tarball?

Comment (cbrueffer) [reply recipes/r-heatmap.plus/meta.yaml]: No.

**Pull request merged**